SETUP.md

```diff
@@ -25,7 +25,7 @@ We recommend using [Visual Studio](https://visualstudio.microsoft.com/vs/), and
 
 To simplify the setup process we provide a [Docker Compose](https://docs.docker.com/compose/) application model. This is split up into multiple service profiles to facilitate easily customization.
 
-Some settings can be customized by modifying the `dev/.env` file, such as the `MSSQL_PASSWORD` which should be modified before starting the project.
+Some settings can be customized by modifying the `dev/.env` file, such as the `MSSQL_PASSWORD` which should be modified before starting the project (Please, comply the [password complexity requirements](https://docs.microsoft.com/en-us/sql/relational-databases/security/password-policy?view=sql-server-ver15#password-complexity), otherwise you'd need to recreate the volume if you put a weak password).
```

Comment:
```suggestion
Some settings can be customized by modifying the `dev/.env` file, such as the `MSSQL_PASSWORD` which should be modified before starting the project. You must comply with the [password complexity requirements](https://docs.microsoft.com/en-us/sql/relational-databases/security/password-policy?view=sql-server-ver15#password-complexity), otherwise your password will be rejected when you try to log in. (If this occurs, you must delete and recreate the volume using a password that meets the complexity requirements.)
```